docs/site/installation.md

```diff
@@ -1,9 +1,9 @@
 ## Installing the prerequisites
 
 ```shell-session
-$ apt install git gcc g++ make python-dev libxml2-dev libxslt1-dev zlib1g-dev gettext curl wget redis-server
-$ wget -q --no-check-certificate -O- https://bootstrap.pypa.io/get-pip.py | sudo python
-$ pip install virtualenv
+$ apt update
+$ apt install git gcc g++ make python3-dev libxml2-dev libxslt1-dev zlib1g-dev gettext curl wget redis-server
+$ wget -q --no-check-certificate -O- https://bootstrap.pypa.io/get-pip.py | sudo python3
```

Comment:
Can you change this to install `python3-pip` from `apt` instead?

Reply:
Done